views/src/app/Unauth/components/login/Login.js

```diff
@@ -18,6 +18,7 @@ const LoginForm = props => {
                 localStorage.token = res.data.token;
                 localStorage.username = res.data.username;
                 history.push('/profile');
+                loginHandler();
             } else {
                 console.log("FAIL log in"); //TODO
```

Comment:
You can use [`console.error()`](https://developer.mozilla.org/en-US/docs/Web/API/Console/error) if you want a more detailed error message.

Reply:
Thank you! I didn't know about this! Gonna use it from now on.